frontend/src/pages/profile/user.js

```diff
@@ -95,80 +104,87 @@ const UserProfile = () => {
-                    }} isInline size="sm" ouiaId="edit-profile-button">Edit</Button></DataListCell>
-                  ]}
-                />
-              }
-              {isEditing &&
-                <DataListItemCells
-                  dataListCells={[
-                    <DataListCell key={1} width={2}><strong>Name:</strong></DataListCell>,
-                    <DataListCell key={2} width={4}>
-                      <InputGroup>
-                        <InputGroupItem isFill ><TextInput value={tempName} type="text" onChange={(_event, value) => setTempName(value)} aria-label="User name" /></InputGroupItem>
-                        <InputGroupItem>
-                          <Button variant="control" icon={<CheckIcon />} onClick={onSaveButtonClicked} ouiaId="edit-save-button">
-                            Save
-                          </Button>
-                        </InputGroupItem>
-                        <InputGroupItem>
-                          <Button variant="control" icon={<TimesIcon />} onClick={() => setIsEditing(false)} ouiaId="edit-cancel-button">
-                            Cancel
-                          </Button>
-                        </InputGroupItem>
-                      </InputGroup>
-                    </DataListCell>
-                  ]}
-                />
-              }
-            </DataListItemRow>
-          </DataListItem>
-          <DataListItem aria-labelledby="E-mail">
-            <DataListItemRow>
-              <DataListItemCells
-                dataListCells={[
-                  <DataListCell key={1} width={2}><strong>E-mail:</strong></DataListCell>,
-                  <DataListCell key={2} width={4}>{user.email}</DataListCell>
-                ]}
-              />
-            </DataListItemRow>
-          </DataListItem>
-          <DataListItem aria-labelledby="Projects">
-            <DataListItemRow>
-              <DataListItemCells
-                dataListCells={[
-                  <DataListCell key={1} width={2}><strong>My Projects:</strong></DataListCell>,
-                  <DataListCell key={2} width={4} style={{paddingTop: 0, paddingBottom: 0}}>
-                    <DataList aria-label="projects" style={{borderTop: 'none'}}>
-                      {projects &&
-                        projects.map((project) => (
-                          <DataListCell key={project.name} className="pf-u-p-sm">
-                            <span> {project.title} </span>
-                            {project.owner_id === user.id &&
-                                  <Label className="project-owner-label" variant="filled" color="green" isCompact>Owner</Label>
-                            }
-                          </DataListCell>
-                        ))
+        {isError && <Alert variant="danger" title="Error fetching user details" />}
+        {!isError &&
+          <Card>
+            <CardBody>
+              {!user && <Skeleton></Skeleton>}
```

Comment:
I think we can continue to improve UX here by extending Skeleton to other components - and getting more fine grained in it's application.

In this case, each `DataListItemCell` could be swapped with Skeleton, so that the rows and "label" cells are rendered showing a skeleton cell.